docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/pipeline-design-and-implementation-supporting-the-workflows.md

```diff
@@ -17,45 +17,45 @@
 1. [Configurations to support working with Feature branches](#configurations-to-support-working-with-feature-branches)
 
     * [User Build setup](#user-build-setup)
-  
+
     * [Pipeline build of feature branches](#pipeline-build-of-feature-branches)
-  
+
     * [Package and Deploy a feature for testing in controlled test environments](#package-and-deploy-a-feature-for-testing-in-controlled-test-environments)
-  
+
 2. [House keeping recommendations](#house-keeping-recommendations)
 
 3. [The Build pipeline for main, epic and release branches](#the-build-pipeline-for-main-epic-and-release-branches)
 
     * [Build pipeline: Build and Test step](#build-pipeline-build-and-test-step)
-  
+
     * [Build pipeline: Install outputs to a shared DEV-TEST runtime](#build-pipeline-install-outputs-to-a-shared-dev-test-runtime)
-  
+
     * [Build pipeline: Analyze step](#build-pipeline-analyze-step)
-  
+
 4. [The Release pipeline: Build, package and deploy](#the-release-pipeline-build-package-and-deploy)
 
     * [Release pipeline: Build stage](#release-pipeline-build-stage)
-  
+
     * [Release pipeline: Packaging stage](#release-pipeline-packaging-stage)
-  
+
     * [Release pipeline: Deployment stage](#release-pipeline-deployment-stage)
-  
+
 5. [Deployment to production](#deployment-to-production)
 
 [Conclusion](conclusion.md#conclusion)
 
 [Appendix](appendix.md#appendix)
 
 
-# Pipeline design and implementation supporting the workflows 
+# Pipeline design and implementation supporting the workflows
 
 In this section, the technical implementation of the different composition of pipeline of the outlined development methodology will be detailed. If a workflow demands a specific configuration, it will be covered within the same section.
 
 ## Configurations to support working with Feature branches
 
 ![](images/media/image16.png) When the developers start working on a new task, they will first create a feature branch. Feature branches are created off the latest code state of the state of the source configuration -- the *main*, *epic* or *release* maintenance branch.
```

Comment:
could we do with a line break between the image and the paragraph here?

Reply:
yes... I think so:
<img width="960" alt="image" src="https://github.com/IBM/dbb-zappbuild/assets/1266441/97263d47-7e33-4c08-bc41-d744fcd303b3">